.github/workflows/tests.yml

```diff
@@ -160,16 +160,21 @@ jobs:
       run: brew readall --aliases
 
     - name: Run brew style on homebrew-core
+      if: matrix.os == 'macOS-latest'
+      run: brew style --display-cop-names homebrew/core
+
+    - name: Run brew style on linuxbrew-core
+      if: matrix.os == 'ubuntu-latest'
       run: brew style --display-cop-names homebrew/core
 
     - name: Run brew style on official taps
       run: brew style --display-cop-names homebrew/bundle homebrew/services homebrew/test-bot
 
-    - name: Run brew cask style
+    - name: Run brew cask style on all taps
```

Comment:
Does it actually run on all taps or just `homebrew/cask`?

Reply:
It seems to run on all taps.

Reply:
Being able to run just on `homebrew/cask` and with `--display-cop-names` would be nice.

Reply:
What I meant is: Does it also run on all other *official* cask taps?

Reply:
Seems like none of them are explicitly tapped before this.

Reply:
It runs on all official and unofficial cask taps and cask fixtures in Homebrew/brew. That's why we had to add a `brew untap` above: https://github.com/Homebrew/brew/blob/068568e6e05757ac33a6bbba8d8d2852e020a4c4/.github/workflows/tests.yml#L107-L108

Reply:
> It runs on all official

But are all official ones tapped though on CI?

Reply:
No, I wouldn't don't think so.